Boyadjie/Uncover
# feat(landing): add story section

## 🤔 Why?

This section is here to display a user story for Unvoy, following the design of the landing page.

## 💻 How?

I added a component `Story.tsx`and updated the bloc `TextAndImg.tsx`.

## ✅ How to validate this PR?

- Check the style
- Check the responsive
- Check orthograph
